dscensor/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM python:3.11.1-slim-buster
+
+# install gcc and other build requirements
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends build-essential && \
+    apt-get install -y --no-install-recommends locales && \
+    rm -rf /var/lib/apt/lists/*
+
+COPY dscensor /app
+COPY INSTALL /app
+COPY LICENSE /app
+COPY MANIFEST.in /app
+COPY openapi /app
+COPY README.md /app
+COPY requirements.txt /app
+COPY setup.cfg /app
+COPY setup.py /app
+COPY autocontent /app
+
+WORKDIR /app
```

Comment:
A bit pedantic; if you set `WORKDIR` before copying then the `COPY` target is just `./`. This is advantageous because if you ever decide to change `WORKDIR` for some reason then you don't have to change all of the `COPY` lines as well.